build/protoc-go/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.13
     unzip -o protoc-3.13.0-linux-x86_64.zip -d /usr/local include/* && \
```

Comment:
you should probably go to `v3.15.8` of `protoc` while you're at it

At the top you could also change to `FROM golang:1.16`